### PR TITLE
Add DisplaySize configuration for 64x32 displays

### PR DESCRIPTION
**Pull request by carlossless**
- [x] Check that you've added documentation to any new methods
- [x] Rebase from `master` if you're not already up to date
- [x] Add or modify an example if there are changes to the public API
- [x] Add a `CHANGELOG.md` entry in the **Unreleased** section under the appropriate heading (**Added**, **Fixed**, **Changed**, etc)
- [x] Run `rustfmt` on the project with `cargo fmt --all` - CI will not pass without this step
- [x] Check that your branch is up to date with master and that CI is passing once the PR is opened

## PR description

Adding a predefined configuration for 64x32 displays, commonly marketed as "0.49" OLED display" like [this panel](https://www.buydisplay.com/0-49-inch-oled-display-module-64x32-pixel-ssd1306-spi-i2c-white-on-black) or [this module](https://www.buydisplay.com/i2c-white-0-49-inch-oled-display-module-64x32-arduino-raspberry-pi).

### CODE REVIEW

**Comment by carlossless** [comment]
@jamwaffles seems like the build is failing, but it's not due to the changes in this PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 ### Added
 
 - [#161](https://github.com/jamwaffles/ssd1306/pull/161) Added a `set_mirror` method to enable or disable display mirroring.
+- [#166](https://github.com/jamwaffles/ssd1306/pull/166) Added DisplaySize configuration for 64x32 displays
```
Comment (jamwaffles) [inline CHANGELOG.md:12]:
```suggestion
- [#166](https://github.com/jamwaffles/ssd1306/pull/166) Added `DisplaySize` configuration for 64x32 displays
```

**Review by jamwaffles — CHANGES_REQUESTED** [review]
A very pedantic nit, otherwise LGTM. I don't have a display that size to test on so can I just confirm that `OFFSETX` should indeed be `32` and isn't a copypasta error?

Thanks for exercising the build too. I pushed a couple of fixes to the `master` branch, so if you could update this branch it should build again.

**Comment by carlossless** [comment]
@jamwaffles oh yes, I have the given configuration running in "production".



For additional reference, other drivers have implemented the same offset: https://github.com/olikraus/u8g2/blob/master/csrc/u8x8_d_ssd1306_64x32.c#L159

**Review by jamwaffles — APPROVED** [review]
Sweet, thanks for confirming!

**Comment by jamwaffles** [comment]
Released in [0.7.1](https://crates.io/crates/ssd1306/0.7.1).

**Pull request merged**